### ISSUE TITLE
Update to flow 0.39

### ISSUE THREAD

**Issue by sleexyz** [clarus/redux-ship#16]
I'm running flow 0.39, and I get the following error just with `yarn add redux-ship`

```
node_modules/redux-ship/src/ship.js:62
 62: ): Ship<Effect, Commit, State, any[]> {
                                    ^^^^^ array type. Only tuples and array literals with known elements can flow to
 79: ) => Ship<Effect, Commit, State, [A1, A2]> =
                                      ^^^^^^^^ tuple type

node_modules/redux-ship/src/ship.js:62
 62: ): Ship<Effect, Commit, State, any[]> {
                                    ^^^^^ array type. Only tuples and array literals with known elements can flow to
 86: ) => Ship<Effect, Commit, State, [A1, A2, A3]> =
                                      ^^^^^^^^^^^^ tuple type

node_modules/redux-ship/src/ship.js:62
 62: ): Ship<Effect, Commit, State, any[]> {
                                    ^^^^^ array type. Only tuples and array literals with known elements can flow to
 94: ) => Ship<Effect, Commit, State, [A1, A2, A3, A4]> =
                                      ^^^^^^^^^^^^^^^^ tuple type

node_modules/redux-ship/src/ship.js:62
 62: ): Ship<Effect, Commit, State, any[]> {
                                    ^^^^^ array type. Only tuples and array literals with known elements can flow to
103: ) => Ship<Effect, Commit, State, [A1, A2, A3, A4, A5]> =
                                      ^^^^^^^^^^^^^^^^^^^^ tuple type

node_modules/redux-ship/src/ship.js:62
 62: ): Ship<Effect, Commit, State, any[]> {
                                    ^^^^^ array type. Only tuples and array literals with known elements can flow to
113: ) => Ship<Effect, Commit, State, [A1, A2, A3, A4, A5, A6]> =
                                      ^^^^^^^^^^^^^^^^^^^^^^^^ tuple type

node_modules/redux-ship/src/ship.js:62
 62: ): Ship<Effect, Commit, State, any[]> {
                                    ^^^^^ array type. Only tuples and array literals with known elements can flow to
124: ) => Ship<Effect, Commit, State, [A1, A2, A3, A4, A5, A6, A7]> =
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ tuple type
```

**Comment by sleexyz** [comment]
It seem like as flow `0.38` tuple arity is now enforced https://github.com/facebook/flow/blob/master/Changelog.md#v0380

**Comment by clarus** [comment]
Closed by #17
